docs/learning/solutions/auto-incident-kubernetes-logs.md

```diff
@@ -0,0 +1,67 @@
+4. Upload the sample job to your Rundeck Enterprise instance by navigating to the **Jobs** tab, selecting **Job Actions** in the upper-right, then selecting **Upload Definition**. 
+You can find more detailed instructions for upload a Job Definition [here](/manual/creating-jobs.html#importing-job-definitions).
+<br><br>![Upload Job](@assets/img/solutions-pd-diag-k8s-upload-job.png)<br><br>
+5. Edit the Job by clicking **Edit This Job**:
+<br><br>![Edit Job](@assets/img/solutions-pd-diag-k8s-edit-job.png)<br><br>
+6. Click into the **Workflow** tab and then in the **Options** section, select the `k8s_selector` option, and modify the selector to determine which pods to pull logs from:
+<br><br>![Edit Selector](@assets/img/solutions-pd-diag-k8s-selector.png)<br>
+<br>![Edit Selector2](@assets/img/solutions-pd-diag-k8s-modify-selector.png)<br><br>
+7. Click into **Step 2** ("Post Logs to PagerDuty").  Click the **Select** button next to **API Key** to select your API Key from Key Storage. If you used a _User Token API Key_ for PagerDuty, then be sure to modify the email-address as well:
+<br><br>![Edit Job2](@assets/img/solutions-pd-diag-k8s-step-2.png)<br><br>
+8. Click **Save** on the step as well as **Save** on the Job.
+
+::: tip Note
+This Rundeck Job is meant to be invoked from PagerDuty, not through the Rundeck GUI. There is a hidden Job Option for the PagerDuty Incident ID. If you run the Job directly from the Rundeck Interface, the Job will fail on Step 2, as it is expecting to have the PagerDuty incident ID as an input parameter.
+:::                                                                                                                                                                                                                                           
+                                                        
+#### Configure Rundeck Actions 
+1. Create a Rundeck User API Token by navigating to **User Icon** -> **Profile** and click the **+** next to **User API Tokens**:
+<br><br>![RD Token](@assets/img/solutions-pd-diag-k8s-rd-token.png)
+2. Enter a **Name** for the API Token and choose a **Role** that has the correct levels of permissions to invoke the uploaded Job.                                                                                                                                                                                                       
+By default, the maximum allowed duration for a Rundeck API Token (before it expires) is 30 days. 
+To optionally remove this expiration, so that API Tokens never expire, add a **Custom Config** property with `rundeck.api.tokens.duration.max`as the property name and set the value to `0`:
```

Comment:
I don't think we should recommend this as it reduces the default security.  It's painful, but that's a choice they should make.